src/pagopa.ts

```diff
@@ -41,7 +43,7 @@ export const AmountInEuroCentsFromNumber = new t.Type<
 
 const PAYMENT_NOTICE_NUMBER_LENGTH = 18;
 
-const MIN_QR_CODE_LENGTH = 44;
+const MIN_QR_CODE_LENGTH = 43;
```

Comment:
`// 44 by specs, see comment above on MIN_AMOUNT_DIGITS`

Reply:
Since _This PR allows to parse amounts consisting of a single digit._ the total length will decrease by 1 digit

`PAGOPA|002|322201151398574181|81005750021|01` -> (44, right decoded)

`PAGOPA|002|322201151398574181|81005750021|1` -> (43, right decoded with these changes)